hw/ip/i2c/doc/_index.md

```diff
@@ -142,6 +148,34 @@ Typically every byte transmitted must also receive an ACK signal, and the IP wil
 However, there are some I2C commands which do not require an ACK.
 In those cases this flag should be asserted with FBYTE indicating no ACK is expected and no interrupt should be raised if the ACK is not received. 
 
+### Target Address Registers
+
+I2C target device is assigned two 7-bit address and 7-bit mask pairs.
+The target device accepts a transaction if the result of the bitwise AND operation performed on the transaction address sent by the host and a mask matches the assigned address corresponding to the mask.
+Thus, with the masks set to all ones (0x7F), the target device will respond to either of the two assigned unique addresses and no other.
+If the mask and the assigned address both have zeros in a particular bit position, that bit will be a match regardless of the value of that bit received from the host.
+Note that if, in any bit position, the mask has zero and the assigned address has one, no transaction can match and such mask/address pair is effectively diasabled.
+The assigned address and mask pairs are set in registers {{< regref TARGET_ID0.ADDRESS >}}, {{< regref TARGET_ID0.MASK >}}, {{< regref TARGET_ID1.ADDRESS >}}, and {{< regref TARGET_ID1.MASK >}}. 
+
+### Acquired Formatted Data
+
+This section applies to I2C in the target mode.
+When the target accepts a transaction, it inserts the transaction address, read/write bit, and START signal sent by the host into ACQ FIFO where they can be accessed by software.
+ACQ FIFO output corresponds to {{< regref ACQDATA >}}.
+If the transaction is a write operation (R/W bit = 0), the target proceeds to read bytes from the bus and insert them into ACQ FIFO until the host terminates the transaction by sending a STOP or RESTART signal.
+A STOP indicator is inserted into ACQ FIFO as the next entry following the last byte received, in which case other bits are junk.
+A RESTART is indicated by inserting both STOP and START indicators into ACQ FIFO as the next entry; the remaining bits are junk.
```

Comment:
A restart is technically different, with unique functionality than a STOP+START, (as I believe @mdhayter pointed out in a previous PR)  Couldn't we just put in a START, and let firmware properly identify it as a RESTART?  This would have the advantage of being simpler to implement.

Reply:
A START indicator alone will be interpreted as a beginning of an accepted transaction. Bits 7:0 of the entry will then be considered the target address and r/w bit.

Setting STOP and START indicators in the same entry is not equivalent to STOP+START. STOP and START will be recorded in two different entries in ACQ FIFO. However, when STOP and START are set in the same entry, this indicates RESTART.

Reply:
I see.   This is probably not a big deal, but you are changing the semantics from I2C symbols.

Reply:
Or we use the same logic but make this a 2-bit enum field:
00 - standard
01 - STOP
10 - START
11 - Repeated START

Probably how the code will end up using it, and avoids the complicated explanation.

Reply:
@mdhayter: I like that idea.